sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -99,6 +99,8 @@ export class BatchingReceiver extends MessageReceiver {
             this.name,
             error
           );
+        } else {
+          error = new Error("An error occurred while receiving messages.");
```

Comment:
Prior to this PR, this error was a `MessagingError`. We are now changing the it to `Error`. Is this intentional? Same for a few lines below.

Reply:
Yeah, it was Error originally and I had changed it to MessagingError, reverted it back.

Reply:
In the master branch it is `MessagingError` ...

Reply:
Oh oops, must have been an intermediary change on local - updated